perception/navigator_vision/nodes/database_color.py

```diff
@@ -75,19 +83,23 @@ def _push(self, data):
             self._images.append(data)
             self._images = self._images[-self._history_length:]
 
-    def get_around_time(self, time, margin=.05):
+    def get_around_time(self, time, margin=.5, timeout=.5):
```

Comment:
Should probably use [message filters](http://wiki.ros.org/message_filters#Example_.28Python.29-1) for this.